docs/adding_tz_env.md

```diff
@@ -0,0 +1,55 @@
+# Proposal: adding TZ as an environment variable 
+
+- Author(s):  [Zhexuan Yang](www.github.com/zhexuany)
+- Last updated:  2018/09/09
+- Discussion at: Not applicable 
+
+## Abstract
+
+When it comes to time-related calculation, it is hard for the distributed system. This proposal tries to resolve two problems: 1. timezone may be inconsistent across multiple `TiDB` instances, 2. performance degradation casued by pushing `System` down to `TiKV`. The impact of this proposal is changing the way of `TiDB` inferring system's timezone name. Before this proposal, the default timezone name pushed down to TiKV is `System` when session's timezone is not set. After this, TiDB evaluates system's timezone name via `TZ` environment variable and then pushes the real name down to `TiKV` rather than `System`.
```

Comment:
If the global timezone is set to UTC and the session's timezone is not set explicitly, we will use UTC.
> the default timezone name pushed down to TiKV is `System` when session's timezone is not set

Reply:
s/evaluates/infers/